docs/sources/view-and-analyze-profile-data/profile-cli.md

```diff
@@ -271,3 +271,37 @@ By default it looks for samples within the last hour, though this can be control
        115366240: 107 13 14 15 16 17 1 2 3
      ...
      ```
+
+### Exporting a profile for Go PGO
+
+You can use the `profilecli query go-pgo` command to retrieve an aggregated profile from a Pyroscope server for use with Go PGO.
+Profiles retrieved with `profilecli query merge` include all samples found in the profile store, resulting in a large profile size, which may cause issues with network transfer and slow down the PGO process.
+In contrast, profiles retrieved with `profilecli query go-pgo` include only the information used in Go PGO, making them significantly smaller and more efficient to handle.
+By default, it looks for samples within the last hour, though this can be controlled with the `--from` and `--to` flags. The source data can be narrowed down with the `--query` flag in the same way as with the `query` command.
+
+1. Specify optional flags.
+
+    - You can provide a label selector using the `--query` flag, for example, `--query='{service_name="my_application_name"}'`.
```

Comment:
We should check if the four spaces renders correctly in the preview build.

Reply:
I confirm it's rendered okay